physx/src/cooking.rs

```diff
@@ -97,7 +100,7 @@ impl Cooking {
         }
     }
 
-    pub fn make_geometry(&mut self, desc: ColliderDesc) -> PhysicsGeometry {
+    pub fn make_geometry(&mut self, desc: ColliderDesc, mesh_scale: Vec3) -> PhysicsGeometry {
```

Comment:
I'm thinking the `mesh_scale` go into the ColliderDesc for TriMesh instead, since it's only used for a single type of geometry, as none of the other ones *can* be scaled to my knowledge. Seems like a crazy source of bugs otherwise. :)

Reply:
That's a much better idea